crates/common/node/src/peer.rs

```diff
@@ -0,0 +1,25 @@
+#[derive(Debug)]
+pub struct PeerCountResponse {
+    pub data: PeerCountData,
+}
+
+#[derive(Debug)]
+pub struct PeerCountData {
+    pub disconnected: String,
+    pub connecting: String,
+    pub connected: String,
+    pub disconnecting: String,
```

Comment:
```suggestion
    #[serde(with = "serde_utils::quoted_u64")]
    pub disconnected: u64,
    #[serde(with = "serde_utils::quoted_u64")]    
    pub connecting: u64,
    #[serde(with = "serde_utils::quoted_u64")]    
    pub connected: u64,
    #[serde(with = "serde_utils::quoted_u64")]    
    pub disconnecting: u64,
```

we should serialize these numbers to strings like this